examples/api/lib/widgets/overscroll_indicator/glowing_overscroll_indicator.2.dart

```diff
@@ -0,0 +1,69 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for GlowingOverscrollIndicator
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+
+// Override the [default drag behavior](https://flutter.dev/docs/release/breaking-changes/default-scroll-behavior-drag).
+class MyCustomScrollBehavior extends MaterialScrollBehavior {
+  @override
+   Set<PointerDeviceKind> get dragDevices => <PointerDeviceKind>{
+     PointerDeviceKind.touch,
+     PointerDeviceKind.mouse,
+  };
+}
+
+void main() => runApp(const MyApp());
+
+class MyApp extends StatelessWidget {
+  const MyApp({Key? key}) : super(key: key);
+
+  static const String _title = 'Flutter Code Sample';
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      scrollBehavior: MyCustomScrollBehavior(),
+      theme: ThemeData(
+        // Set [TargetPlatform](https://api.flutter.dev/flutter/foundation/TargetPlatform-class.html)
```

Comment:
If this is necessary then we should explain why. Otherwise, it doesn't seem relevant to the example.